app/assets/stylesheets/application.scss

```diff
@@ -1,53 +1,77 @@
 //= require_tree .
-
 @import "bootstrap-sprockets";
 @import "bootstrap";
 @import "bootstrap-datetimepicker";
 @import "footer";
 
-// Add fonts
-@import url(https://fonts.googleapis.com/css?family=Poppins:300,400,500,600,700&subset=latin-ext);
+@import url(https://fonts.googleapis.com/css?family=Roboto:400,100,500,900);
+
 @import "font-awesome-sprockets";
 @import "font-awesome";
 
+$btncolor: #FF9062;
+$btnshadow: #EC5357;
+$panelcolor: #5EB4AF;
+
 html{
   height: 100%;
 }
 
 body {
-  font-family: "Poppins", sans-serif;
+  font-family: "Roboto", sans-serif;
```

Comment:
If you're using variables in CSS already, you could always create:

$font = "Roboto"

But that's just a small workflow thing.